src/calls/list-vehicles.ts

```diff
@@ -36,7 +40,11 @@ const locationSchema = z.object({
   timestamp: z.string(),
   signal_source: z.enum(['Gps', 'Gsm']),
   accuracy_radius: z.number().optional(),
-});
+}).transform(data => ({
+  inMovement: data.in_movement,
+  signalSource: data.signal_source,
+  accuracyRadius: data.accuracy_radius
+}))
```

Comment:
```suggestion
}).transform(({ in_movement, signal_source, accuracy_radius, ...data => ({
  ...data,
  inMovement: data.in_movement,
  signalSource: data.signal_source,
  accuracyRadius: data.accuracy_radius
}))
```

This schema should be moved to the `shared.ts` file and reused in `list-vehicles.ts` and `get-equipment.ts`